tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/species_features.tsx

```diff
@@ -14,3 +14,434 @@ export const feature_mcolor3: Feature<string> = {
   component: FeatureColorInput,
 };
 
+export const flavor_text: Feature<string> = {
+  name: "Flavor Text",
+  description: "Describe your character!",
+  component: FeatureTextInput,
+};
+
+export const ooc_notes: Feature<string> = {
+  name: "OOC Notes",
+  description: "Talk about your character OOCly!",
+  component: FeatureTextInput,
+};
+
+export const custom_species: Feature<string> = {
+  name: "Custom Species Name",
+  description: "Want to have a fancy species name? Put it here, or leave it blank.",
+  component: FeatureShortTextInput,
+};
+
+export const custom_species_lore: Feature<string> = {
+  name: "Custom Species Lore",
+  description: "Add some lore for your species! Won't show up if there's no custom species.",
+  component: FeatureTextInput,
+};
+
+export const allow_mismatched_parts_toggle: FeatureToggle = {
+  name: "Allow Mismatched Parts",
+  description: "Want to go completely crazy with your character design? Use this to select any parts from any species!",
+  component: CheckboxInput,
+};
+
+export const mutant_colors_color: Feature<string[]> = {
+  name: "Mutant Colors",
+  description: "Legacy colors for controlling shit.",
+  component: FeatureTriColorInput,
+};
+
+export const body_markings_toggle: FeatureToggle = {
+  name: "Body Markings",
+  description: "Add some lore for your species! Won't show up if there's no custom species.",
+  component: CheckboxInput,
+};
+
+export const feature_body_markings: Feature<string> = {
+  name: "Body Markings Selection",
+  description: "Want to have a fancy species name? Put it here, or leave it blank.",
```

Comment:
Lmao, copy-paste moment. All of the `description` under this have been missed, whoops.